examples/prediction/xgboost-high-level-apis.ipynb

```diff
@@ -29,16 +29,100 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
It's better to keep the null outputs, user can run that directly. And I think no need to update `examples/prediction/trained_ames_model.dat`.

Reply:
done